inventory_reporting/reports/inventory_report.py

```diff
@@ -337,3 +351,55 @@ def generate_consumable_report(self, workbook, wizard):
                     16,
                     valuation.stock_move_id.analytic_account_names or "",
                 )
+
+    def generate_summary_report(self, workbook, wizard):
+        ws = workbook.add_worksheet(_("Inventory Report Summary"))
+        valuation_obj = self.env["stock.valuation.layer"]
+        product_categories = self.get_product_categories()
+        base_storable_domain, storable_categories = self.get_storable_categories(wizard)
+        headers = [
+            _("Product Category"),
+            _("SVL's Total Inventory Value"),
+            _("Inventory Operation Type"),
+            _("SVL's Total Inventory Value"),
+        ]
+        column_widths = [30, 20, 30, 30]
```

Comment:
```suggestion
        column_widths = [30, 20, 30, 20]
```
Why did you switch the 4th column from 30 to 20? The current pattern doesn't make sense to me.

Reply:
I think the column names were long so Lin-san spread them out. I've put it back.
Done

Reply:
@AungKoKoLin1997 It often doesn't make good design sense to follow the label length. We should first evaluate based on the type of data and its length.

Reply:
@yostashiro I see. I didn't want to overlap the header label but I will follow up on this practice in the future.